ui/components/multichain/account-list-menu/account-list-menu.tsx

```diff
@@ -520,6 +544,19 @@ export const AccountListMenu = ({
               </Box>
               ///: END:ONLY_INCLUDE_IF
             }
+            {isAddWatchEthereumAccountEnabled && (
+              <Box marginTop={4}>
+                <ButtonLink
+                  disabled={!isAddWatchEthereumAccountEnabled}
+                  size={ButtonLinkSize.Sm}
+                  startIconName={IconName.Eye}
+                  onClick={handleAddWatchAccount}
+                  data-testid="multichain-account-menu-popover-add-watch-only-account"
```

Comment:
should the test ids be set to constants somewhere?

Reply:
I don't think there is a constants file in the extension for test IDs, but I will do some digging!

Reply:
fair enough.